kroxylicious-test-tools/src/main/java/io/kroxylicious/proxy/config/ClusterNetworkAddressConfigProviderDefinitionBuilder.java

```diff
@@ -17,7 +18,14 @@ public ClusterNetworkAddressConfigProviderDefinitionBuilder(String type) {
 
     @Override
     protected ClusterNetworkAddressConfigProviderDefinition buildInternal(String type, Map<String, Object> config) {
-        var configType = ClusterNetworkAddressConfigProviderContributorManager.getInstance().getConfigType(type);
+        Class<? extends BaseConfig> result;
+        try {
+            result = ContributionManager.INSTANCE.getDefinition(ClusterNetworkAddressConfigProviderContributor.class, type, (clazz, typeName) -> "").configurationType();
+        }
+        catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException("No endpoint provider found for name '" + type + "'");
```

Comment:
should this catch block be replaced by the `getDefinition` errorGenerator?

Reply:
yeah.

Reply:
fixed in ec1c649b79f516f46e01d0991948a8bc05556932